tests/snuba/api/endpoints/test_organization_events_v2.py

```diff
@@ -1954,3 +1956,44 @@ def test_deleted_issue_in_results(self):
             assert len(data) == 2
             assert data[0]["issue"] == event1.group.qualified_short_id
             assert data[1]["issue"] == "unknown"
+
+    def test_histogram_function(self):
+        self.login_as(user=self.user)
+        project = self.create_project()
+        start = before_now(minutes=2).replace(microsecond=0)
+        latencies = [
+            (1, 999, 5),
+            (1000, 1999, 4),
+            (3000, 3999, 3),
+            (6000, 6999, 2),
+            (10000, 10000, 1),  # just to make the math easy
+        ]
+        for bucket in latencies:
+            for i in range(bucket[2]):
+                milliseconds = random.randint(bucket[0], bucket[1])
+                data = load_data("transaction")
+                data["transaction"] = "/error_rate/{}".format(milliseconds)
+                data["timestamp"] = iso_format(start)
+                data["start_timestamp"] = iso_format(start - timedelta(milliseconds=milliseconds))
+                self.store_event(data, project_id=project.id)
+
+        with self.feature(
+            {"organizations:discover-basic": True, "organizations:global-views": True}
+        ):
+            response = self.client.get(
+                self.url,
+                format="json",
+                data={
+                    "field": ["histogram(transaction.duration, 10)", "count()"],
+                    "query": "event.type:transaction",
+                    "sort": "histogram_transaction_duration_10",
+                },
+            )
+
+            assert response.status_code == 200, response.content
+            data = response.data["data"]
+            assert len(data) == 5
```

Comment:
@evanh I'm curious. Why do only 5 buckets show up when 10 are requested in `histogram(transaction.duration, 10)`?

Reply:
It's because the other 5 buckets are empty. Right now the function only returns buckets that have values.

Reply:
@evanh would it be possible to include those empty buckets? buckets with `0` counts? 😬 

Asking because I'll have to reconstruct these empty buckets to normalize the data set for the graph.

Reply:
Yeah I figured you might ask that. Thought I'd try without just to see if it was something the frontend already handled.